reversion_compare/compare.py

```diff
@@ -22,6 +23,18 @@
 logger = logging.getLogger(__name__)
 
 
+@python_2_unicode_compatible
+class FieldVersionDoesNotExist(object):
+    """
+    Sentinel object to handle missing fields
+    """
+
+    def __str__(self):
+        return force_text(ugettext("Field Didn't exist!"))
```

Comment:
I preserved the old translation here, but I agree this would be a good change to do later.